pkg/providers/amifamily/al2.go

```diff
@@ -63,12 +64,20 @@ func (a AL2) DescribeImageQuery(ctx context.Context, ssmProvider ssm.Provider, k
 			IsMutable: amiVersion == v1.AliasVersionLatest,
 		})
 		if err != nil {
+			errs = append(errs, err)
 			continue
 		}
 		ids[imageID] = variants
 	}
 	// Failed to discover any AMIs, we should short circuit AMI discovery
 	if len(ids) == 0 {
+		if len(errs) > 0 {
+			return DescribeImageQuery{}, fmt.Errorf(
```

Comment:
Would it be possible to return something like a `DescribeImageError(err)` that either returns either the line on :81 or a wrapped error? Then we'd just be able to return that in all files.